worksheet_A/TerminalHacking.cpp

```diff
@@ -1,46 +1,107 @@
-// PartA_TerminalHacking.cpp : Defines the entry point for the console application.
-//
-
-#include "stdafx.h"
-#include "WordList.h"
-
-const int wordLength = 5;
-const int numberOfWords = 15;
-
-int main()
-{
-	// Seed the random number generator with the current time,
-	// to ensure different results each time the program is run
-	srand(static_cast<unsigned int>(time(nullptr)));
-
-	// Initialise word list
-	WordList words(wordLength);
-
-	// Choose secret word
-	std::string secret = words.getRandomWord();
-
-	// Create a set to hold the list of options
-	std::set<std::string> options;
-
-	// Put the secret word in the set
-	options.insert(secret);
-
-	// Fill the set with more words
-	// Using a set for options guarantees that the elements are all different
-	while (options.size() < numberOfWords)
-	{
-		std::string word = words.getRandomWord();
-		options.insert(word);
-	}
-
-	// Display all words
-	for each (std::string word in options)
-	{
-		std::cout << word << std::endl;
-	}
-
-	// TODO: implement the rest of the game
-
-    return 0;
-}
-
+// PartA_TerminalHacking.cpp : Defines the entry point for the console application.
+//
+
+#include "stdafx.h"
+#include "WordList.h"
+
+const int wordLength = 5;
+const int numberOfWords = 15;
+
+int main()
+{
+	// Seed the random number generator with the current time,
+	// to ensure different results each time the program is run
+	srand(static_cast<unsigned int>(time(nullptr)));
+
+	// Initialise word list
+	WordList words(wordLength);
+
+	// Choose secret word
+	std::string secret = words.getRandomWord();
+
+	// Create a set to hold the list of options
+	std::set<std::string> options;
+
+	// Put the secret word in the set
+	options.insert(secret);
+
+	// Fill the set with more words
+	// Using a set for options guarantees that the elements are all different
+	while (options.size() < numberOfWords)
+	{
+		std::string word = words.getRandomWord();
+		options.insert(word);
+	}
+
+	// Display all words
+	for each (std::string word in options)
+	{
+		std::cout << word << std::endl;
+	}
+
+	// TODO: implement the rest of the game
+	
+	string guessedWord;
+	int likeness = 0;
+	int guesses = 0;
+	
+	//<------------------------------------------------------------
+	//Probably won't work here but is working code.
+	cout << "\nHow many letters do you want per word? 4-7\n";
+    	cin>>opt; //Fetches user response
+         
+         switch (opt) //selection statement - option

```

Comment:
This will only work if `opt` has type `char`. If you want to read an integer as input, you can simply do `cin >> wordLength` and the conversion will happen automatically. Or if you want to convert a string to an integer, look up the `std::stoi` function.

As a general rule, if you're writing a big repetitive `switch`-`case` block like this, there's often a simpler and better approach!